app/javascript/gobierto_data/webapp/components/commons/DownloadButton.vue

```diff
@@ -9,33 +9,34 @@
       class="gobierto-data-btn-download-data"
       @click.native="isHidden = !isHidden"
     />
-    <transition
-      name="fade"
-      mode="out-in"
-    >
-      <div
-        v-show="!isHidden"
-        class="gobierto-data-btn-download-data-modal"
+    <keep-alive>
+      <transition
+        name="fade"
+        mode="out-in"
       >
         <div
-          v-for="(item, key, index) in arrayFormats"
-          :key="index"
-          :item="item"
+          v-show="!isHidden"
+          class="gobierto-data-btn-download-data-modal"
         >
-          <a
-            :href="editor ? sqlfileCSV : item"
-            :download="titleDataset"
-            class="gobierto-data-btn-download-data-modal-element"
+          <div
```

Comment:
You don't need to use a `div` tag to wrap for-loops. If it's not necessary, you may wanna use `<template v-for....>`